app/scripts/views/settings/display_name.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'cocktail',
+  'views/base',
+  'views/form',
+  'lib/auth-errors',
+  'stache!templates/settings/display_name',
+  'views/mixins/settings-mixin',
+  'views/mixins/settings-panel-mixin',
+  'views/mixins/avatar-mixin'
+],
+function (Cocktail, BaseView, FormView, AuthErrors, Template,
+  SettingsMixin, SettingsPanelMixin, AvatarMixin) {
+  var t = BaseView.t;
+
+  var View = FormView.extend({
+    template: Template,
+    className: 'display-name',
+
+    events: {
+      'click .settings-unit-toggle': '_openSettingsUnit'
+    },
+
+    _openSettingsUnit: function () {
+      this.navigate('/settings/display_name');
+    },
+
+    context: function () {
+      return {
+        displayName: this._displayName
```

Comment:
Is it easier to fetch this directly from the account/user instead of storing a local variable that could get out of sync?